.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: go
+dist: xenial
```

Comment:
I wasn't aware that xenial is available! Is that recent?...

Reply:
No, it is around 2 years old (http://releases.ubuntu.com/xenial/)